app/javascript/dashboard/components/ChatList.vue

```diff
@@ -24,18 +24,27 @@
       <div class="filter--actions">
         <div v-if="!hasAppliedFiltersOrActiveFolders" class="filter--wrap">
           <assignee-type-filter
+            v-if="!isAllConversation"
             :items="assigneeTabItems"
+            :active-tab="activeAssigneeTab"
             class="chat-status--filter"
             @filterChange="updateAssigneeTab"
           />
           <chat-filter
             class="chat-status--filter"
+            :active-type="activeStatus"
             @statusFilterChange="updateStatusType"
           />
-          <sort-by-filter @changeSortByFilter="updateSortByFilterType" />
+          <sort-by-filter
+            :active-type="activeSortBy"
+            @changeSortByFilter="updateSortByFilterType"
```

Comment:
@iamsivin Can we change the prop name on filter components to `change` from `changeSortByFilter`? As they are already in the component scope, and there aren't multiple on change events.

Reply:
Please do this for othet components as well